recipes/coloc_sat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "coloc_sat" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf81bc02612caf603000530b174343cd8a3076e9ad1e5f8a4600e182c0601600
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - Coloc_between_product_and_mission = coloc_sat.scripts.coloc_between_product_and_mission:main
+    - Coloc_2_products = coloc_sat.scripts.coloc_2_products:main
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
```

Comment:
It looks like you're missing `setuptools-scm` from the `host` section, which I believe will mess up the version of the package.

Reply:
@synapticarbors 
So I have to add "-  setuptools_scm" in the host section, right?
Tell me if I am wrong but I don't know if it is necessary because I already posted packages without specifying this in meta.yaml ( such as https://github.com/conda-forge/xarray-safe-s1-feedstock ) and it didn't have version problems. When I am looking to the documentation of meta.yaml, I don't see this mention for setuptools-scm ( https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html ).

Reply:
It's specified in the `build-system`:
https://github.com/umr-lops/coloc_sat/blob/main/pyproject.toml#L23-L25

The link to the conda docs only shows an example. I just know that we've seen issues before when it is left out. It may depend on how it's being used. See for example:
https://github.com/conda-forge/staged-recipes/pull/23598#issuecomment-1675142623

Reply:
So yes, you'd put it under the `host` requirements. Also when I run `grayskull` to generate the recipe for this package it specifies including `setuptools-scm`.

Reply:
See also:
https://conda-forge.org/docs/maintainer/knowledge_base.html#using-setuptools-scm

I don't think you need to add anything beyond including it under `host`.

Reply:
@synapticarbors Okay, thank you for your help! I just made this change